superset/utils.py

```diff
@@ -323,6 +322,11 @@ def base_json_conv(obj):
         return str(obj)
     elif isinstance(obj, timedelta):
         return str(obj)
+    elif isinstance(obj, bytes):
+        try:
+            return '{}'.format(obj)
```

Comment:
This will return different values depending on Python version. For Python 2 (where bytes are strings):

```python
>>> print(repr('{}'.format(b'test')))
'test'
```

For Python 3:

```python
>>> print(repr('{}'.format(b'test')))
"b'test'"
```

Why not return a list of integers here instead?

```python
>>> list(b'test')
[116, 101, 115, 116]
```

What is the use case for `VARBINARY`?